src/cli_util.rs

```diff
@@ -2008,7 +2008,7 @@ pub fn get_config_file_path(
     let edit_path = match config_source {
         // TODO(#531): Special-case for editors that can't handle viewing directories?
         ConfigSource::User => {
-            config_path()?.ok_or_else(|| user_error("No repo config path found to edit"))?
+            new_config_path()?.ok_or_else(|| user_error("No repo config path found to edit"))?
```

Comment:
nit: let's rename `get_config_file_path()` to something like `get_new_config_file_path()` to prevent future misuse.